modules/tensor_mechanics/test/include/functions/InitialStrikeShearStress.h

```diff
@@ -0,0 +1,52 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+/*
+Define Function for Initial Shear Stress along Strike Direction
+Problem-Specific: TPV205-2D
```

Comment:
what is TPV205-2D?

Reply:
This is a benchmark problem from USGS. https://strike.scec.org/cvws/tpv205_210docs.html